### ISSUE TITLE
Haven't been able to update HS Deck Tracker for a year

### ISSUE THREAD

**Issue by Yupppi** [HearthSim/Hearthstone-Deck-Tracker#3707]
My problem is, when I start the deck tracker, the prompt to update to the new version pops up, but when I try to install it, I'll face an error message. Anyhow, the manual updating does work. The command prompt gives a following message (sorry for the command prompt being on finnish OS):

![deck tracker 1](https://user-images.githubusercontent.com/24705202/41723608-2f5cb1da-7574-11e8-9680-717c2b9fd8d0.PNG)

![deck tracker 2](https://user-images.githubusercontent.com/24705202/41723617-3302fa88-7574-11e8-87dd-e86bdd37a3c7.PNG)

It's a problem with downloading the update? If I try to launch the HDTUpdate.exe manually, it just flashes the command prompt and closes. Maybe firewall? But I don't know how to allow the updater from the firewall because it's the software's prompt opening a new thing. I tried to allow the updater in multiple ways but it didn't change the outcome.

**Comment by jleclanche** [comment]
I'm seeing TLS issues. That indeed sounds like it could be firewall/antivirus interfering. Have you tried temporarily disabling the firewall and seeing if it's still happening? If it does then it's something else.

You could also have the firewall whitelist hsdecktracker.net, githubusercontent.com and amazonaws.com. That might help.

**Comment by Yupppi** [comment]
Got the same result with firewall disaled and after allowing the three domains (even though I don't think they were blocked by the firewall).

**Comment by riQQ** [comment]
Duplicate of #3587

**Comment by Yupppi** [comment]
Thanks, it did work. Will it keep working in the future or will I have to do this every time since installing the new version never worked automatically?

**Comment by riQQ** [comment]
The new version is fixed and should update correctly.
